apps/api/src/domain/models/media-item/entities/media-item.validate-invariants.spec.ts

```diff
@@ -20,10 +22,50 @@ describe(`MediaItem.validateInvariants`, () => {
 
                     assertErrorAsExpected(
                         result,
-                        new InvariantValidationError(mediaItem.getCompositeIdentifier(), [])
+                        new InvariantValidationError(mediaItem.getCompositeIdentifier(), [
+                            new InconsistentMediaItemPropertyError(
+                                'lengthMilliseconds',
+                                inconsistentMimeType
+                            ),
+                        ])
                     );
                 });
             });
         });
     });
+
+    // TODO Do we inclue videos?
```

Comment:
Tag this comment with [this story](https://www.pivotaltracker.com/story/show/186725983).

Reply:
done